staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -336,6 +353,15 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.ShutdownWatchTerminationGracePeriod, "shutdown-watch-termination-grace-period", s.ShutdownWatchTerminationGracePeriod, ""+
 		"This option, if set, represents the maximum amount of grace period the apiserver will wait "+
 		"for active watch request(s) to drain during the graceful server shutdown window.")
+}
 
-	utilfeature.DefaultMutableFeatureGate.AddFlag(fs)
```

Comment:
Seems like a mistake to lose this.  Given we take a featuregate, how about adding the featuregate flag here so we end up with an encapsulated entity?